src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -398,6 +397,8 @@ public void hoverClickAndViewPhotoOnTableCell(int questionBodyIndex, int tableRo
          * regarding unix systems and current testing version of Selenium and Firefox
          */
         JavascriptExecutor jsExecutor = (JavascriptExecutor) browser.driver;
+        jsExecutor.executeScript("arguments[0].scrollIntoView(true); window.scrollBy(0, -200);",
+                                           browser.driver.findElement(By.id(idOfQuestionBody)));
```

Comment:
How does this solve the problem? Can we abstract the technique so that it can be used in other places?